arcade/gui/widgets/dropdown.py

```diff
@@ -87,13 +91,37 @@ def _update_options(self):
             )
             button.on_click = self._on_option_click
 
+    def _find_ui_manager(self):
+        # search tree for uimanager
+
+        parent = self.parent
+        while isinstance(parent, UIWidget):
+            parent = parent.parent
+
+        return parent if isinstance(parent, UIManager) else None
+
+    def _show_overlay(self):
+        manager = self._find_ui_manager()
+        if manager is None:
+            raise Exception("UIDropdown could not find ")
```

Comment:
The Exception message looks incomplete.

```suggestion
            raise Exception("UIDropdown could not find UIManager in its parents.")
```
The more correct word maybe ancestor, but for when beginners encounter this error, parents seems better.